src/include/daos_fs.h

```diff
@@ -1122,6 +1121,27 @@ dfs_removexattr(dfs_t *dfs, dfs_obj_t *obj, const char *name);
 int
 dfs_listxattr(dfs_t *dfs, dfs_obj_t *obj, char *list, daos_size_t *size);
 
+
+enum {
+	DFS_CHECK_PRINT		= (1 << 0),
+	DFS_CHECK_REMOVE	= (1 << 1),
+	DFS_CHECK_LINK_LF	= (1 << 2),
+};
+
+/**
+ * Scan the DFS namespace and check if there are any leaked objects. Add those object to
+ * "Lost+Found" directory.
+ *
+ * \param[in]	dfs	Pointer to the mounted file system.
```

Comment:
This looks like it's out of date with the code and will need fixing so the Doxygen is inline with the code.
```suggestion
 * \param[in]	poh	Open pool handle
```